demos/mqtt/mqtt_demo_mutual_auth/mqtt_demo_mutual_auth.c

```diff
@@ -93,6 +93,18 @@
     #define NETWORK_BUFFER_SIZE    ( 1024U )
 #endif
 
+#ifndef OS_NAME
+    #define OS_NAME    "aws-iot-device-sdk-embedded-C"
+#endif
+
+#ifndef OS_VERSION
+    #define OS_VERSION    "4.0.0"
```

Comment:
Do we want to call it OS_VERSION or SDK_VERSION ? Same for OS_NAME ...

Reply:
OS_VERSION because we want to emphasize the operating system aspect like outlined in the workdocs I sent you. Although I do use the sdk name here for now. When these metrics go to FreeRTOS.org it will be replaced with "FreeRTOS".

Reply:
Per offline convo we are updating to SDK_